packages/macros/package.json

```diff
@@ -14,6 +14,11 @@
   "license": "MIT",
   "author": "Edward Faulkner",
   "main": "src/index.js",
+  "exports": {
+    ".": "./src/index.js",
+    "./babel": "./src/babel.js",
+    "./src/*": "./src/*.js"
```

Comment:
Do we want people to import everything from `src`? Or is this for backwards compatibility, because people were doing so before?

Reply:
its backwards compatibility because _we_ were doing it before.
I didn't audit what all we're importing, but it probably would be good to restrict at some point.

Reply:
if we're going to restrict it now is the time. We won't be able to reduce it once this is landed because we really don't want a new major. Can we just remove it and see what fails?

Reply:
I'm more than happy to do a follow-up PR which removes `src`.
I don't really want to make this PR do 2 things (1. current focus, 2. change available modules)

Reply:
I agree with both of you: let's remove it, but in a follow-up PR :)